main.go

```diff
@@ -40,8 +59,16 @@ func update(path string) {
 		// Get all files changed without extension
 		for _, v := range outS {
 			split := strings.Split(v, "/")
-			last := split[len(split)-1]
-			filesChanged = append(filesChanged, strings.Split(last, ".")[0])
+			if top {
+				first := split[0]
+				filesChanged = append(filesChanged, first)
+			} else {
+				last := split[len(split)-1]
+				filesChanged = append(filesChanged, strings.Split(last, ".")[0])
+			}
+		}
+		if top {
+			filesChanged = removeDuplicates(filesChanged)
```

Comment:
@nikitavoloboev Maybe we should run this command regardless if `--top` is passed in?

Reply:
agree, lets do that

Reply:
done 👍